### ISSUE TITLE
Add possibility to add tab based on content type

### ISSUE THREAD

**Issue by alexander-schranz** [sulu/sulu#3044]
| Q | A
| --- | ---
| Bug? | no
| New Feature? | yes
| Sulu Version | 1.4

#### Actual Behavior

Currently its not possible to add in the DisplayOption of a ContentNavigationProvider that a Tab is only visible when a specific content type is part of the template. Currently it is only based on template name.

#### Expected Behavior

When creating a bundle like the SuluFormBundle a tab should be able to register everywhere a specific content type is used.

#### Possible Solutions

Addd a new DisplayOption class which will add this behaviour:

```php
$item->setDisplayConditions(
    [
        new DisplayTypeCondition('form_select'),
    ]
);
```

**Comment by danrot** [comment]
That's an interesting approach, we should probably look into that.

**Comment by danrot** [comment]
@alexander-schranz Haven't that been solved?

**Comment by alexander-schranz** [comment]
@danrot currently not in the form bundle it need to be configured where a tab should be displayed.

**Comment by danrot** [comment]
I think we solved that in the 2.0 refactoring. Reopen with an explanation if I am wrong :see_no_evil:
